[Armis Event Collector] Fix initial fetch request (#30005)
demisto/content: Packs/Armis/ReleaseNotes/1_1_2.md

#### Integrations

##### Armis Event Collector

> - Fixed an issue where the initial fetch request returned old events instead of current time events.

Reviewer: ```suggestion
- Fixed an issue that caused the initial fetch request to return old events instead of current ones.
```
